docs/_source/tutorials/notebooks/labelling_textclassification_setfit_zeroshot.ipynb

```diff
@@ -0,0 +1,645 @@
+              "                                                text  \\\n",
+              "0                           How do I locate my card?   \n",
+              "1  I still have not received my new card, I order...   \n",
+              "2  I ordered a card but it has not arrived. Help ...   \n",
+              "3   Is there a way to know when my card will arrive?   \n",
+              "4                       My card has not arrived yet.   \n",
+              "\n",
+              "                                             vectors  \n",
+              "0  {'mini-lm-sentence-transformers': [-0.01016701...  \n",
+              "1  {'mini-lm-sentence-transformers': [-0.04284121...  \n",
+              "2  {'mini-lm-sentence-transformers': [-0.03365556...  \n",
+              "3  {'mini-lm-sentence-transformers': [0.012195922...  \n",
+              "4  {'mini-lm-sentence-transformers': [-0.04361867...  "
+            ]
+          },
+          "execution_count": 7,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "dataset.to_pandas().head()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0-_YRm_GhaDT"
+      },
+      "source": [
+        "## 🔫 Zero-shot predictions with SetFit\n",
+        "\n",
+        "The original `banking77` dataset is a intent classification dataset with dozens of labels (`lost_card`, `card_arrival`, etc.). To keep this tutorial simple, we define a simplified labelling scheme with higher level classes.\n",
+        "\n",
+        "\n",
+        "Let's set up and train our zero-shot SetFit model. Please note that SetFit's approach to zero-shot is to create a synthetic dataset of training examples, which is different from other approaches (e.g., transformers zero-shot pipelines) where \"templated\" examples with label names are used at inference time."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "7LVpVjNtheHG"
+      },
+      "outputs": [],
+      "source": [
+        "labels = [\"change details\", \"card\", \"atm\", \"top up\", \"balance\", \"transfer\", \"exchange rate\", \"pin\"]\n",
+        "train_dataset = get_templated_dataset(\n",
+        "    candidate_labels=labels, \n",
+        "    sample_size=8, \n",
+        "    template=\"The customer request is about {}\"\n",
+        ")\n",
+        "\n",
+        "model = SetFitModel.from_pretrained(\"all-MiniLM-L6-v2\")\n",
+        "trainer = SetFitTrainer(\n",
+        "    model=model,\n",
+        "    train_dataset=train_dataset\n",
+        ")\n",
+        "trainer.train()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Sp3oSMi9snkp"
+      },
+      "source": [
+        "Let's use our trained model to predict over the dataset. We'll later load this predictions into Argilla and use them to speed-up the labelling process."
```

Comment:
```suggestion
        "We can use our trained zero-shot model to predict over the dataset. We'll later load these predictions into Argilla and use them to speed-up the labelling process."
```
This one is a bit nitpicky. I think it's good to include `zero-shot` for clarity, and `Let's` was used 3 times in a row, so I changed it up too. `this` was incorrect.